specification/monitor/data-plane/Microsoft.Insights/preview/2023-03-01-preview/metricBatch.json

```diff
@@ -0,0 +1,449 @@
+      "type": "object",
+      "properties": {
+        "additionalInfo": {
+          "description": "Additional information about the error",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "description": "The type of the info property (e.g. string).",
+                "type": "string"
+              },
+              "info": {
+                "description": "Additional information related to the error.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "code": {
+          "description": "Error code",
+          "type": "string"
+        },
+        "message": {
+          "description": "Error message indicating why the operation failed.",
+          "type": "string"
+        }
+      }
+    },
+    "Response": {
+      "description": "The metrics result for a resource.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "timespan": {
+            "type": "string",
+            "description": "The timespan for which the data was retrieved. Its value consists of two datetimes concatenated, separated by '/'.  This may be adjusted in the future and returned back from what was originally requested."
+          },
+          "interval": {
+            "type": "string",
+            "format": "duration",
+            "description": "The interval (window size) for which the metric data was returned in.  This may be adjusted in the future and returned back from what was originally requested.  This is not present if a metadata request was made."
+          },
+          "namespace": {
+            "type": "string",
+            "description": "The namespace of the metrics been queried"
+          },
+          "resourceregion": {
+            "type": "string",
+            "description": "The region of the resource been queried for metrics."
+          },
+          "resourceid": {
+            "type": "string",
+            "description": "The resource that has been queried for metrics."
+          },
+          "value": {
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/Metric"
+            },
+            "description": "the value of the collection."
+          }
+        },
+        "required": [
+          "timespan",
+          "value"
+        ]
+      }
+    },
+    "MetricBatchBody": {
+      "description": "List of resource ids to fetch metric namespaces for",
+      "type": "array",
+      "items": {
+        "type": "string"
+      },
+      "example": [
+        "/subscriptions/123456728-abcd-1234-abcd-123456789abc/resourceGroups/TestGroup/providers/Microsoft.Storage/storageAccounts/TestStorage1",
+        "/subscriptions/123456728-abcd-1234-abcd-123456789abc/resourceGroups/TestGroup/providers/Microsoft.Storage/storageAccounts/TestStorage2"
+      ]
+    },
```

Comment:
This schema appears to be unused and has the same basic shape as `ResourceIdList`. Can this one be deleted?